src/molecules/formfields/CreditCardField/__tests__/credit_card_field.spec.js

```diff
@@ -7,7 +7,7 @@ import CreditCardField from 'molecules/formfields/CreditCardField';
 describe('<CreditCardField />', () => {
 
   it('renders', () => {
-    const wrapper = mount(<CreditCardField input={{}} />);
+    const wrapper = mount(<CreditCardField input={{}} meta={{}} />);
```

Comment:
I think you also need to add `meta` filed in the `Readme.md`. I got an error when I spun it up locally